src/java.base/share/classes/sun/nio/ch/FileChannelImpl.java

```diff
@@ -538,7 +538,7 @@ public void force(boolean metaData) throws IOException {
     // Assume at first that the underlying kernel supports sendfile();
     // set this to false if we find out later that it doesn't
     //
-    private static volatile boolean transferSupported = true;
+    private static volatile boolean transferToSupported = true;
```

Comment:
At some point we need to invert these to transferXXXNotSupported so that the default is false rather true, maybe we should do it as part of this PR because all usages are changed anyway?